Ten Elephants/Ten Elephants/Models/GenericCodingKeys.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CodingKeys.swift
+//  Ten Elephants
+//
+//  Created by Дарья Домрачева on 09.12.2021.
+//
+
+import Foundation
+
+public class GenericCodingKeys: CodingKey {
```

Comment:
Тут и ниже не нужна публичность
```suggestion
class GenericCodingKeys: CodingKey {
```